docs-conceptual/azps-6.1.0/install-az-ps.md

```diff
@@ -25,6 +25,7 @@ Az PowerShell modules, and makes their cmdlets available for use.
 > [!NOTE]
 > PowerShell 7.x and later is the recommended version of PowerShell for use with the Azure Az
 > PowerShell module on all platforms.
+> You need to run powershell with `Run as Administrator` option in order to be able to enable and import Azure Az module.
```

Comment:
Running PowerShell elevated as an admin isn't required to install or run the Az PowerShell module. Our recommended installation option is the `CurrentUser` scope.